spec/factories/competitions.rb

```diff
@@ -0,0 +1,13 @@
+FactoryGirl.define do
+  factory :competition do
+    code_name { Faker::Lorem.word }
+    start_date { Faker::Date.between(10.days.from_now, 20.days.from_now) }
```

Comment:
Default to being in the middle of competiton - start_date in past, end_date in future